x-pack/plugin/security/src/main/bin/elasticsearch-security-config.bat

```diff
@@ -11,7 +11,7 @@ setlocal enableextensions
 set ES_MAIN_CLASS=org.elasticsearch.xpack.security.cli.ConfigInitialNode
 set ES_ADDITIONAL_SOURCES=x-pack-env;x-pack-security-env
 set ES_ADDITIONAL_CLASSPATH_DIRECTORIES=lib/tools/security-cli
-call "%~dp0elasticsearch-cli.bat -strict" ^
+call "%~dp0elasticsearch-cli.bat" "-strict" ^
```

Comment:
bug fix, unrelated to the changes of this PR. Irrelevant as this CLI tool is going to be removed as part of #77231